dev-tools/README.md

```diff
@@ -0,0 +1,20 @@
+# Development Setup for Anchore Engine
+
+*Note: These directions are for PyCharm only.*
+
+## Dev Dependencies:
+
+In the same environment as your project interpereter, run `pip3 install -r requirements-dev.txt`
+
+## File Watchers
+Please import the file watchers in watchers.xml to use with the [File Watchers Pycharm Plugin](https://plugins.jetbrains.com/plugin/7177-file-watchers).
+
+Steps to import:
```

Comment:
Please add the Pycharm version  you used for this - at some point JetBrains will (unfortunately) move where this is configured at.

Reply:
done